bevy_rapier3d/Cargo.toml

```diff
@@ -7,38 +7,50 @@ documentation = "http://docs.rs/bevy_rapier3d"
 homepage = "http://rapier.rs"
 repository = "https://github.com/dimforge/bevy_rapier"
 readme = "../README.md"
-keywords = [ "physics", "dynamics", "rigid", "real-time", "joints" ]
+keywords = ["physics", "dynamics", "rigid", "real-time", "joints"]
 license = "Apache-2.0"
 edition = "2021"
 
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [lib]
 path = "../src/lib.rs"
-required-features = [ "dim3" ]
+required-features = ["dim3"]
 
 [features]
-default = [ "dim3", "async-collider", "debug-render-3d" ]
+default = ["dim3", "async-collider", "debug-render-3d"]
 dim3 = []
-debug-render = [ "debug-render-3d" ]
-debug-render-2d = [ "bevy/bevy_core_pipeline", "bevy/bevy_sprite", "bevy/bevy_gizmos", "rapier3d/debug-render", "bevy/bevy_asset" ]
-debug-render-3d = [ "bevy/bevy_core_pipeline", "bevy/bevy_pbr", "bevy/bevy_gizmos", "rapier3d/debug-render", "bevy/bevy_asset" ]
-parallel = [ "rapier3d/parallel" ]
-simd-stable = [ "rapier3d/simd-stable" ]
-simd-nightly = [ "rapier3d/simd-nightly" ]
-wasm-bindgen = [ "rapier3d/wasm-bindgen" ]
-serde-serialize = [ "rapier3d/serde-serialize", "bevy/serialize", "serde" ]
-enhanced-determinism = [ "rapier3d/enhanced-determinism" ]
-headless = [ ]
-async-collider = [ "bevy/bevy_asset", "bevy/bevy_scene" ]
+debug-render = ["debug-render-3d"]
+debug-render-2d = [
+    "bevy/bevy_core_pipeline",
+    "bevy/bevy_sprite",
+    "bevy/bevy_gizmos",
+    "rapier3d/debug-render",
+    "bevy/bevy_asset",
+]
+debug-render-3d = [
+    "bevy/bevy_core_pipeline",
+    "bevy/bevy_pbr",
+    "bevy/bevy_gizmos",
+    "rapier3d/debug-render",
+    "bevy/bevy_asset",
+]
```

Comment:
Nitpick: These changes are welcome ❤️, but out of scope and adds up to a lot of noise for an already non trivial Pull Request. It's not blocking but an ideal situation would be to isolate them in another PR, or not include them.